addons/web/static/src/views/kanban/kanban_cover_image_dialog.xml

```diff
@@ -27,7 +27,7 @@
                     onUpload.bind="onUpload"
                     resModel="props.record.resModel"
                     resId="props.record.resId">
-                        <button class="btn btn-primary" t-on-click="uploadImage">
+                        <button class="btn btn-primary w-auto" t-on-click="uploadImage">
```

Comment:
Addressing this issue here is not appropriate because the width of the footer button in the web_enterprise module is set at 45%.